### PR TITLE
Added tag titles for all pages

### PR DESCRIPTION
**Pull request by ybalenko**
## What

Added page tag titles for all the pages.

## Why Do

> Per designer's request [All tag titles are listed under the main prototype in Figma](https://www.figma.com/file/vs1l6ERmi4AqYjxBBK9gDJ/Style-Guide-%26-Prototypes?type=design&node-id=2432%3A31484&mode=design&t=gllZ1DH08vO9N8Tw-1).

## Show Me

> Checked it locally.

Screenshot 2023-11-08 at 13 51 30

### CODE REVIEW

**Review by jnakaso — CHANGES_REQUESTED** [review]
Hard coding the names of the projects will not scale.  I suggest using the projectService to get the title from Sanity.  The `project_individual.tsx` page has code that pull in a project.

**Comment by jnakaso** [comment]
> Hard coding the names of the projects will not scale. I suggest using the projectService to get the title from Sanity. The `project_individual.tsx` page has code that pull in a project.

Something like:

`  const lookupTitle = async (pathName: string): Promise<string> => {
    const path = pathName.split('/')
    const pageName = path[path.length - 1]
    switch (pageName) {
      case 'project_individual':
        const project = searchParams.get('project')
        const p = await dasProjectsService.getOne(project)
        return p ? p.title : DEFAULT_TAG
      case 'volunteer_individual':
        return 'TBD'  // When role pages come online
      default:
        const tag = TAG_NAMES[pageName]
        return tag ?? DEFAULT_TAG
    }
    
      useEffect(() => {
    if (pathName) {
      lookupTitle(pathName)
        .then(t => setTitle(t));
    }
  }, [pathName])
  
  `

**Comment by jnakaso** [comment]
OBTW, I have found that

const params = new URLSearchParams(window.location.search)

works better than nextJS' useSearchParams

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,14 +1,61 @@
...
+  about: 'About us | Digital Aid Seattle',
+  projects: 'Projects | Digital Aid Seattle',
+  privacy: 'Privacy policy | Digital Aid Seattle',
+}
 
 export default function App({ Component, pageProps }: AppProps) {
+  const pathName = usePathname()
+  const searchParams = useSearchParams()
+
+  let pageName = null
```
Comment (jnakaso) [inline pages/_app.tsx:27]: Just a recommendation, but to limit the scope of variables, `pageName` should be moved within the lookupTitle function.   the useEffect hook would depend on the pathName.

Limiting the scope of variables is a good engineering practice.

**Review by jnakaso — APPROVED** [review]
Looks good.  See my comment about coding practices.

File: pages/_app.tsx (hunk 1 above)
Comment (ybalenko) [reply pages/_app.tsx]: Thank you, Jeff, for the feedback. I've refactored the code and tested it locally.

**Pull request merged**